src/main/java/blackjack/domain/player/Player.java

```diff
@@ -0,0 +1,77 @@
+package blackjack.domain.player;
+
+import blackjack.domain.card.Card;
+import blackjack.domain.Status;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public abstract class Player {
+    protected static final int ACE_CRITICAL_POINT = 11;
+    protected static final int ACE_UPPER_POINT = 11;
+    protected static final int ACE_LOWER_POINT = 1;
+    protected static final int BLACKJACK_SCORE = 21;
+    protected static final int START_INDEX = 0;
+
+    protected List<Card> cards = new ArrayList<>();
+    protected String name;
+    protected Status status;
+
+    public void addCard(Card card) {
```

Comment:
외부에서 CardDeck에서 카드를 뽑아서 넘겨주지 말고 Player가 직접 CardDeck에게 요청해 카드를 얻도록 해보면 어떨까요? 
```suggestion
    public void addCard(CardDeck deck) {
        this.cards.add(deck.pop());
    }
```

Reply:
수정했습니다!